urbanstats-persistent-data/urbanstats_persistent_data/main.py

```diff
@@ -64,50 +65,67 @@ def juxtastat_register_user_request():
 
     print("RECV", form)
 
-    if "user" in form and "domain" in form:
-        register_user(form["user"], form["domain"])
-        return flask.jsonify(dict(success=True))
-    return flask.jsonify({"error": "Needs parameters user and domain!"}), 200
+    if "user" in form and "secureID" in form and "domain" in form:
+        registration_error = register_user(
+            form["user"], form["secureID"], form["domain"]
+        )
+        return flask.jsonify(dict(success=True, registration_error=registration_error))
```

Comment:
`success=true` but also there's an error is hella confusing

Reply:
Hmmmm, removed success, which wasn't really being used